### PR TITLE
Update projects

### PR DESCRIPTION
**Pull request by bishabosha**
Links Meriam, also subdivides the project page into categories, rather than active/completed

### CODE REVIEW

**Comment by julienrf** [comment]
Could you post a screenshot of the projects page?

**Comment by bishabosha** [comment]
Screenshot 2020-04-16 at 20 54 10

**Comment by julienrf** [comment]
I’m not sure about classifying by category rather than completion.

**Comment by bishabosha** [comment]
How about categorising only the active ones?

**Comment by julienrf** [comment]
Actually, just _categorising_ seems to be an unnecessary overhead to me.

**Comment by adpi2** [comment]
Actually I think the categorisation is good because it is more meaningful for most people. I mean, most people don't know what a "Tasty reader" is or what "Trait encoder init" is but they would surely be interested in knowing that those are steps towards Scala 3.

Into categories it should be sorted sorted by status though.

**Comment by darjutak** [comment]
Thank you @adpi2, my idea exactly. It can really be confusing to anyone landing on our website when they see a bunch of mixed projects under "active" - what can one do with that info?

Projects are categorized in alphabetical order under active now, so the first thing you see is Advisory Board, and Metals are lost somewhere down there, for example... How does that help us promote what we actually do?

By the rate of opening new projects, the overhead is so insignificant it's not worth mentioning, IMO.

Why not try a new proposal and see if it helps a bit?

In conclusion, I am strongly suggesting to keep new categorization.

**Comment by julienrf** [comment]
Thanks for your responses, I’m not opposed to the categorization, go for it!

**Comment by bishabosha** [comment]
One thing for the future, the category division is manual, it should probably be automatic based on the `category` property

**Comment by adpi2** [comment]
The overhead may be decreased by collaboration. Each and everyone of us should be responsible for maintaining its status and projects up to date at a manageable rate.
